docs/asciidoc/configuration.asciidoc

```diff
@@ -420,6 +426,37 @@ IMPORTANT: You must set your <<hosts,hosts>> to the proper hostname _with_ port.
     It may not work setting <<port,port>> and <<hosts,hosts>> to only a host
     name due to the different connection module used.
 
+[[aws_sign_request]]
+=== aws_sign_request
+
+WARNING: This feature has not been fully tested and should be considered BETA.
+
+WARNING: This setting will not work unless the `requests-aws4auth` and `boto3` Python
+    modules have been manually installed first.
```

Comment:
This should probably read, "If installing via `pip`, this setting will not work unless ...".  I install `requests-aws4auth` in the [RPM/DEB](https://github.com/elastic/curator/blob/v5.2.0/unix_packages/build_official_package.sh#L143) packages.  If you want to edit both of the files in `unix_packages` directory to add `boto` to be installed, that would be okay, too.